locales/en.yml

```diff
@@ -119,7 +113,7 @@ profile.objectivesList.teach: Teach together
 profile.objectivesList.project: Dance project
 profile.profileSorts.username: Username
 profile.profileSorts.registered: Registered
-profile.profileSorts.last-seen: Last seen
+profile.profileSorts.lastSeen: Last seen
 profile.placeholder.username: username
 profile.placeholder.whatsapp: whatsapp chat
```

Comment:
it's better when fieldname goes before attribute, i.e. `profile.whatsapp.placeholder`